sqlglot/generator.py

```diff
@@ -262,6 +255,9 @@ def sql(self, expression, key=None):
         if isinstance(expression, exp.Func):
             return self.function_fallback_sql(expression)
 
+        if isinstance(expression, exp.Property):
```

Comment:
why was this needed to be added?

Reply:
If a property-specific generator doesn't exist, it will fall back to the generic property generator. Is there a better way I could achieve this behavior?

Reply:
i thought how it was handled before was that it always called _property_sql and then federated from there?

Reply:
this may be ok for now, but what's changed that it needed to be added is my question